File: tests/fixtures/sample.kt
import kotlin.math.sqrt
import kotlin.collections.List

class Calculator(private val value: Double) {
    fun add(x: Double): Double {
        return value + x
    }

    fun subtract(x: Double): Double {
        return value - x
    }
}

fun main() {
    val calc = Calculator(10.0)
    val result = calc.add(5.0)
    println(result)
}

File: tests/fixtures/sample.rs
use std::collections::HashMap;
use std::io;

struct Calculator {
    value: f64,
}

impl Calculator {
    fn new(v: f64) -> Self {
        Calculator { value: v }
    }

    fn add(&self, x: f64) -> f64 {
        self.value + x
    }
}

fn main() {
    let calc = Calculator::new(10.0);
    let result = calc.add(5.0);
    let mut map = HashMap::new();
    println!("{}", result);
}
